.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: Push Default
+on:
+  # push:
+  #   branches: [ master ]
+  push:
```

Comment:
Don't want to accidentally trigger a push so the it's commented out, will change back to the correct command before merging.

Reply:
Not sure what you meant -- this workflow is already being triggered in this PR because of the auto-build condition you added. It's ok to trigger a push because it's being built with the "auto_build" tag rather than the "latest" tag (which is what is used by default).

Reply:
I have updated to trigger this on master and release.